rasa/shared/core/trackers.py

```diff
@@ -809,6 +814,44 @@ def latest_action_name(self) -> Optional[Text]:
             ACTION_TEXT
         )
 
+    def persist(self, path: Union[Text, pathlib.Path], filename: Text) -> Text:
+        """Saves the tracker to a file.
+
+        Args:
+            path: Directory that the tracker should be saved in.
+            filename: Name of the file to save the tracker in.
+
+        Returns:
+            Full path of the stored tracker file.
+        """
+        tracker_file = pathlib.Path(path) / filename
+        rasa.shared.utils.io.create_directory_for_file(tracker_file)
+
+        # noinspection PyTypeChecker
+        rasa.shared.utils.io.write_text_file(str(jsonpickle.encode(self)), tracker_file)
+
+        return str(tracker_file)
```

Comment:
Do we have a plan about what to do with the result value? I mean, does it have to be `string` (if we just log it). or should we rather return `Path`, in case we want to i.g. concat it later.

Reply:
It's a question, not asking for a change 😄

Reply:
It should be used exactly as in the test. I suppose a Path object would be more general, though, so I'll change it to that.